jhu-idc/idc-isle-dc
# Remove Matomo from stack

Remove Matomo module from composer and associated config; publish new
snapshot image with updated config, and without matomo content.

## To Test

Do `make up` and verify that the site still works.  

Note:  I cherry picked this onto the testcafe branch, and the simple "is the site still up" tests passed.
